backend/selenium_tests/registration_data_import/test_registration_data_import.py

```diff
@@ -174,3 +176,87 @@ def test_smoke_registration_data_import_happy_path(
+    ) -> None:
+        business_area = BusinessArea.objects.get(name="Afghanistan")
+        business_area.kobo_token = "kobo_token"
+        business_area.kobo_username = "hope_kobo_admin_nga"
+        business_area.save()
+
+        # Go to Registration Data Import
+        pageRegistrationDataImport.selectGlobalProgramFilter("Test Programm").click()
+        pageRegistrationDataImport.getNavRegistrationDataImport().click()
+        assert pageRegistrationDataImport.titleText in pageRegistrationDataImport.getPageHeaderTitle().text
+        pageRegistrationDataImport.getButtonImport().click()
+        # Check Elements on Page
+        assert pageRegistrationDataImport.getButtonImportFile().get_property("disabled")
+        pageRegistrationDataImport.getImportTypeSelect().click()
+        assert pageRegistrationDataImport.koboItemText in pageRegistrationDataImport.getKoboItem().text
+        pageRegistrationDataImport.getKoboItem().click()
+        pageRegistrationDataImport.getInputName().send_keys("Test 1234 !")
+
+        # pageRegistrationDataImport.checkLoadingProgressCircle()
+        pageRegistrationDataImport.getKoboProjectSelect().click()
+        pageRegistrationDataImport.select_listbox_element("Education new programme").click()
+        # pageRegistrationDataImport.checkLoadingProgressCircle()
+
+        assert "0" in pageRegistrationDataImport.getNumberOfHouseholds().text
+        assert "0" in pageRegistrationDataImport.getNumberOfIndividuals().text
+        assert pageRegistrationDataImport.buttonImportFileIsEnabled()
+        pageRegistrationDataImport.getButtonImportFile().click()
+        pageRegistrationDataImport.checkAlert("Cannot import empty form")
+
+    @pytest.mark.vcr(ignore_localhost=True, ignore_hosts=["elasticsearch"])
+    def test_import_kobo_form(
+        self,
+        login: None,
+        create_programs: None,
+        pageRegistrationDataImport: RegistrationDataImport,
+        pageDetailsRegistrationDataImport: RDIDetailsPage,
+    ) -> None:
+        business_area = BusinessArea.objects.get(name="Afghanistan")
+        business_area.kobo_token = "kobo_token"
+        business_area.kobo_username = "hope_kobo_admin_nga"
+        business_area.save()
+
+        country = Country.objects.get(name="Afghanistan")
+        business_area.countries.set([country])
+
+        area_type_1 = AreaType.objects.create(name="State", area_level=1, country=country)
+        area_type_2 = AreaType.objects.create(
+            name="Local government area", area_level=2, country=country, parent=area_type_1
+        )
+        area_type_3 = AreaType.objects.create(name="Ward", area_level=3, country=country, parent=area_type_2)
+        area_1 = Area.objects.create(name="Borno", p_code="NG008", area_type=area_type_1)
+        area_2 = Area.objects.create(name="Bama", p_code="NG008003", area_type=area_type_2, parent=area_1)
+        Area.objects.create(name="Andara", p_code="NG008003001", area_type=area_type_3, parent=area_2)
```

Comment:
From 
business_area = BusinessArea.objects.get(name="Afghanistan") 
to
Area.objects.create(name="Andara", p_code="NG008003001", area_type=area_type_3, parent=area_2)

From my perspective that should be in separate pytest fixture/fixtures. Consider for test clearance.